pyproject.toml

```diff
@@ -93,6 +93,10 @@ script = "scripts.test.run:mypy_src"
 help = "Run mypy on 'tests' using the installed stubs"
 script = "scripts.test:test(dist=True, type_checker='mypy')"
 
+[tool.poe.tasks.mypy_coverage]
+help = "Run mypy on 'tests' using the local stubs and report HTML coverage"
+script = "scripts.test:test(coverage=True, type_checker='mypy')"
```

Comment:
Could also make it an optional argument to `poe mypy`

Reply:
Not sure how I could do this.

Reply:
something like this might work (haven't tested it):


```
[tool.poe.tasks.mypy]
help = "Run mypy on 'tests' (using the local stubs) and on the local stubs"
script = "scripts.test:test(src=True, coverage=coverage, type_checker='mypy')"
args = [{name = "coverage", positional = false, default = false, type = "boolean", required = false, help= "Whether to report coverage as HTML"}]
```

Reply:
Thanks.  I've gone this way.  I also reformatted the args into a more friendly TOML format.  Let me know if you don't like this style.

Reply:
nice! I didn't use this style because it created some errors that I couldn't figure out :)

Reply:
I currently can't test it because my python 3.10.7 installation triggers the `error: Positional-only parameters are only supported in Python 3.8 and greater  [syntax]`-bug

Reply:
> I currently can't test it because my python 3.10.7 installation triggers the `error: Positional-only parameters are only supported in Python 3.8 and greater [syntax]`-bug

I think that's an issue with python 3.10.7 and mypy 0.971.  If you use python 3.10.6 (which is what we pinned in the github actions), you should be OK.